scripts/core/editor3/actions/editor3.jsx

```diff
@@ -0,0 +1,21 @@
+export function onChangeEditorState(editorState) {
+    return {
+        type: 'CHANGE_EDITOR_STATE',
```

Comment:
Let's set a standard here for consistency. How about we call all actions prefixed with the component name, so in this case: `EDITOR_CHANGE_STATE`